src/main/java/Wqchat.java

```diff
@@ -0,0 +1,118 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+public class Wqchat {
+    private static Task[] tasks = new Task[100];
+    private static void printLine() {
+        System.out.println("____________________________________________________________");
+    }
+
+    private static void printTaskCount() {
+        if (taskCount == 0) {
+            System.out.println("Now you have 1 task in the list.");
+        } else {
+            int count = taskCount + 1;
+            System.out.println("Now you have " + count + " tasks in the list");
+        }
+    }
+    private static void printGreetings() {
+        printLine();
+        System.out.println("Hello! I'm Wqchat");
+        System.out.println("What can I do for you?");
+        printLine();
+    }
+    private static void printList() {
+        printLine();
+        System.out.println("Here are the tasks in your list: ");
+        for (int i = 0; i < taskCount; i++) {
+            System.out.print(i + 1);
+            System.out.print(".");
+            System.out.println(tasks[i].toString());
+        }
+        printLine();
+    }
+
+    private static void printAddedTask() {
+        printLine();
+        System.out.println("Got it. I've added this task:");
+        System.out.println(tasks[taskCount]);
+        printTaskCount();
+        printLine();
+        taskCount++;
+    }
+    protected static int taskCount = 0;
+    private static final int EVENT_DESCRIPTION_INDEX = 6;
```

Comment:
Good utilisation of constants. Great compliance with the full caps constant naming standards.